### ISSUE TITLE
Making easier to deploy with KO to different namespace

### ISSUE THREAD

**Issue by chmouel** [tektoncd/pipeline#917]
# RFE: Allow KO to deploy to different namespace easily

Currently KO don't allow easily to install to different namespace than the default `'tekton-pipelines' ` which is less than ideal for multi users dev clusters

It really only uses what's inside the config/*.yaml and it references `tekton-pipelines`

Suggestions:

1. Templatize it (with kustomize or simple go template) and pre generate them before running ko. IE: with a make target

2. Extend ko to allow override the namespace (if that could be accepted)

3. Provide a KO wrapper that does this for us

**Comment by bobcatfish** [comment]
Another possible idea: switching to something other than `ko`

@ImJasonH @jonjonsonjr any thoughts?

**Comment by jonjohnsonjr** [comment]
This is possible if you aren't already specifying a namespace, see my comment here: https://github.com/google/go-containerregistry/issues/295

I'd use `ko` in conjunction with another tool, like kustomize or even sed:
```
ko resolve -f config/ | sed 's/tekton-pipelines/other-namespace/g' | kubectl apply -f -
```

**Comment by chmouel** [comment]
Thanks for the tip @jonjohnsonjr I have sent a PR to update our documentation in #972, the sed wasn't as straightforward, but i believe I figured it out to install it properly.

**Comment by dlorenc** [comment]
Closing as fixed by #972
